src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -222,16 +240,22 @@ export default class Pilot {
      * @param {Number} speed - the speed to maintain, in knots
      * @return {Array} [success of operation, readback]
      */
-    maintainSpeed(currentSpeed, speed) {
+    maintainSpeed(currentSpeed, speed, aircraftModel) {
         const instruction = radio_trend('speed', currentSpeed, speed);
+        let speedUnattainable = '';
```

Comment:
same as above
>boolean type properties should be prefixed with the word is and worded in a truthy manner

so this could be revised to `isAssignedSpeedUnattainable`